services/applications_service.js

```diff
@@ -3,11 +3,13 @@ const repo = require('../repositories/applications_repository');
 async function getAllApplications() {
     return await repo.getAllApplications();
 }
+
 async function createNewApplication(name, about, email, video_link, discord_nickname) {
     return await repo.createNewApplication(name, about, email, video_link, discord_nickname);
 }
-async function updateApplicationStatus(id, status, comment, user_id) {
-    return await repo.updateApplicationStatus(id, status, comment, user_id);
+
```

Comment:
I think we need to update swagger here :)

Reply:
this is changed inside API and not recevied from client, so we're fine here. but thanks for pointing that out! 🙌🏻